l10n_br_fiscal/models/document_line.py

```diff
@@ -63,6 +63,14 @@ def _compute_amount(self):
                 record.amount_tax_withholding
             )
 
+            # Amount Estimate Tax
+            if record.product_id.icms_origin in ['1', '2', '6', '7']:
+                record.amount_estimate_tax = \
```

Comment:
Deveria verificar o campo fiscal_type Se ele for um serviço e o tax_icms_or_issqn == TAX_DOMAIN_ISSSQ deve pegar os impostos aproximados do NBS caso contrário deve pegar os impostos do NCM

Reply:
Ok

Reply:
DONE